nservicebus/msmq/transportconfig.md

```diff
@@ -58,29 +58,25 @@ Often when debugging MSMQ using [native tools](viewing-message-content-in-msmq.m
 
 snippet:ApplyLabelToMessages
 
+## Transactions and delivery guarantees
 
-## Controlling transaction scope options
+MSMQ transport supports all [transaction handling modes](/nservicebus/messaging/transactions.md), i.e. Transaction scope, Receive only, Sends atomic with Receive and No transactions.
 
-The following options can be configured when the MSMQ transport is working in the transaction scope mode.
+Refer to [Transport Transactions](/nservicebus/messaging/transactions.md) for detailed explanation of the supported transaction handling modes and available configuration options. 
 
 
-### Isolation level
+### Transaction scope
 
-NServiceBus will by default use the `ReadCommitted` [isolation level](https://msdn.microsoft.com/en-us/library/system.transactions.isolationlevel).
+In this mode the ambient transaction is started before receiving the message. The transaction encompasses all stages of processing including user data access and saga data access. If all the logical data stores (transport, user data, saga data) use the same physical store there is no escalation to Distributed Transaction Coordinator (DTC).
```

Comment:
Nothing specific, but for SQL it's also covered in the transport specific doco. Since it's only for those two I think it makes sense to leave it in those places, instead of putting in the shared article.